src/components/LIcon.vue

```diff
@@ -94,13 +94,11 @@ export default {
       scheduleCreateIcon();
     });
```

Comment:
The reason why I skipped l-icon completely is because is the only truly 'custom' component in my opinion (that does not bind 1:1 with a leaflet one). WDYT?

Reply:
Yeah, I do agree that it is by far the most custom of the components, and `mapObject` was never exposed on it in Vue2Leaflet, so I agree it is certainly fine to leave `leafletObject` out here too!

There is still technically a Leaflet object inside the component that _could_ be exposed at some point, which is why I asked, although it would sometimes be `L.icon` and sometimes `L.divIcon`, and if nobody has asked for it in however many years of Vue2Leaflet anyway...